.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,66 @@
+name: Windows Build
+
+on:
+  push:
+    tags: [ "*" ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: npm
+          cache-dependency-path: |
+            package-lock.json
+            .github/actions/setup-protoc-env/package-lock.json
+
+      - name: Setup Bazelisk
+        uses: bazelbuild/setup-bazelisk@v2
+
+      - name: Mount bazel cache
+        uses: actions/cache@v3
+        with:
+          path: "~/.cache/bazel"
+          key: bazel
+
+      - name: Install protoc
+        uses: arduino/setup-protoc@v1
```

Comment:
I don't think we can use this for security reasons. Can you do something like this instead? 

https://github.com/protocolbuffers/protobuf-javascript/blob/main/.github/workflows/codeql.yml#L61-L68

Reply:
Sure. That will probably make it so I can completely skip the custom action but I'll see.

Reply:
I've now completely removed the arduino/setup-protoc dep and standardized the download of the protoc executable/includes between build.yml and codeql.yml

Should you want to update the version, you can now just copy and paste the top environment variable between the two files easily.

This also simplified the actual codeql.yml config as well into separate steps.